src/buildURL.js

```diff
@@ -19,7 +19,7 @@ const buildURL = {
         let baseUrl = `https://${options.baseSvcUrl}/services/search/FindingService/v1?`;
         baseUrl += 'SECURITY-APPNAME=' + options.clientID;
         baseUrl += '&OPERATION-NAME=' + options.operationName;
-        baseUrl += '&SERVICE-VERSION=1.0.0&RESPONSE-DATA-FORMAT=JSON';
+        baseUrl += '&SERVICE-VERSION=1.0.0';
         baseUrl += options.param ? '&' + options.param + '=' + options.name : '';
```

Comment:
I think this `buildSearchUrl ` is used by many methods, I would suggest we should keep these changes separate. May be we need to make changes everywhere at once.